.travis.yml

```diff
@@ -24,19 +18,31 @@ addons:
 before_install:
   - export CXX="g++-4.8" CC="gcc-4.8"
   - nvm install 6
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
+  - 'export DISPLAY=:99.0'
+  - >-
+    /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid
+    --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0
+    1280x1024x16 -extension RANDR
   - sh -e /etc/init.d/xvfb start
 install:
- - make travissetup
+  - make travissetup
+  - pip install pygments lxml setuptools --upgrade
```

Comment:
If you need a specific version, then explicitly indicate the version, otherwise --upgrade is going to waste time unnecessarily.  If all you need is the bikeshed prerequisites, then let the installation of bikeshed cover that.